docs/_website/generator/generate.go

```diff
@@ -123,12 +124,18 @@ func main() {
 
 	docsRoot := "../../"
 	destRoot := filepath.Join(docsRoot, "_website/generated/docs")
+	progressive := flag.Bool("progressive", false, "Regenerate documentation without removing existing builds")
 
-	// Delete any old files.
-	subdirs, _ := ioutil.ReadDir(destRoot)
-	for _, f := range subdirs {
-		if err := os.RemoveAll(filepath.Join(destRoot, f.Name())); err != nil {
-			log.Fatal(err)
+	flag.Parse()
+
+	// Delete any old files if progressive flag is not passed in.
+	if !*progressive {
+		fmt.Println("Removing existing generated docs...")
```

Comment:
super nit: this is confusing since there's no corresponding output that says generating docs

either delete this message or add a corresponding message to the start of generation

Reply:
👍